recipe/build.sh

```diff
@@ -1,7 +1,32 @@
 #!/usr/bin/env bash
 
 # Install DAGMC
-cmake -DMOAB_DIR="${PREFIX}" \
+# default options from
+# https://github.com/svalinn/DAGMC/blob/develop/cmake/DAGMC_macros.cmake
+
+cmake -DBUILD_MCNP5=OFF
```

Comment:
each of these lines needs a backslash ("\") to continue the line

Reply:
Thanks sorry about that, looks like that might have fixed it